pkg/flow/internal/controller/metrics.go

```diff
@@ -11,18 +13,20 @@ type controllerMetrics struct {
 }
 
 // newControllerMetrics inits the metrics for the components controller
-func newControllerMetrics(r prometheus.Registerer) *controllerMetrics {
+func newControllerMetrics(r prometheus.Registerer, id string) *controllerMetrics {
```

Comment:
Our mixin will need to be updated to have some awareness of modules, can you start thinking about how these should be monitored/visualized?